css/index.css

```diff
@@ -1 +1,352 @@
-/* Compile your LESS file! */
\ No newline at end of file
+.centered {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Excellent use of a mixin here! Extremely popular one is to make a all-around centered mixin :)